topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,30 @@
+package strings
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of runes.
+	runes := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the runes.
+	var beg int
+
+	// Here  we create int that will be a pointer to the end of the runes.
+	end := len(runes) - 1
+
+	// While there are still code points to check.
+	for beg < end {
+
+		// Swap the code point.
```

Comment:
Remove all the line spaces between the three lines of code that follow.

// Perform the swap of runes.

Reply:
Use the word rune not code point